src/Components/Components/test/RenderTreeBuilderTest.cs

```diff
@@ -398,15 +398,15 @@ public void CannotAddAttributeAtRoot()
         }
 
         [Fact]
-        public void CannotAddEventHandlerAttributeAtRoot()
+        public void CannotDelegateAttributeAtRoot()
         {
             // Arrange
             var builder = new RenderTreeBuilder(new TestRenderer());
 
             // Act/Assert
             Assert.Throws<InvalidOperationException>(() =>
             {
-                builder.AddAttribute(0, "name", eventInfo => { });
+                builder.AddAttribute(0, "name", new Action<UIEventArgs>(eventInfo => { }));
```

Comment:
Could you clarify why this change is here? The test name only talks about delegates in general, not specific types of delegates. So why wasn't the previous code sufficient to represent that?

Reply:
This is needed to make the type conversion work. I tried to keep the semantics of what's being tested the same.